### PR TITLE
feat(minidump): Store minidumps in a temp directory for debugging

### PR DESCRIPTION
**Pull request by jan-auer**
This PR stores minidumps in a temporary location on our webservers for debugging purposes. They should be deleted within 24-48 hours.

Fixes SENTRY-6CW

### CODE REVIEW

**Comment by jan-auer** [comment]
@JTCunning I wasn't sure if there's a setting to determine the `/tmp` location, so I hard coded it for now. Maybe it would be better to turn that off during tests and development and only activate it in production - is there something similar we can do or a configuration option that I can reuse for this?

File: src/sentry/web/api.py [hunk 1]
```diff
@@ -54,6 +55,8 @@
 PIXEL = base64.b64decode('R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=')
 
 PROTOCOL_VERSIONS = frozenset(('2.0', '3', '4', '5', '6', '7'))
+MINIDUMP_TMP_PATH = '/tmp/minidump'
```
Comment (mitsuhiko) [inline src/sentry/web/api.py]: can we put this behind a sentry settings key?

**Review by mitsuhiko — APPROVED** [review]

**Review by mitsuhiko — CHANGES_REQUESTED** [review]
approved by accident

**Review by mitsuhiko — APPROVED** [review]

**Comment by jan-auer** [comment]
@JTCunning note that this will require two new configs to be set in prod: 

```py
# Toggles whether minidumps should be cached
SENTRY_MINIDUMP_CACHE = False
# The location for cached minidumps
SENTRY_MINIDUMP_PATH = '/tmp/minidump'
```

**Review by JTCunning — APPROVED** [review]
🍪

**Pull request merged**